src/Builder/ModelInputShaper.cpp

```diff
@@ -55,13 +55,22 @@ void ModelInputShaper::setShapeInformation(
   if (!shapeInformation.empty()) {
     std::stringstream shapeInfoString(shapeInformation);
     std::string shapeString;
+    bool hasAllInputSetting = false;
     while (std::getline(shapeInfoString, shapeString, ',')) {
+      assert(!hasAllInputSetting &&
```

Comment:
Would we possibly create an error as opposed to an assert? Better for user-fronting issues.

Reply:
Yes, will change it to an error.

Reply:
Done. Thanks!